HomeWork/src/main/java/HomeWork5/HomeWork5.java

```diff
@@ -58,7 +57,7 @@ public static String stringReverse (String reverse) {
         return new  StringBuilder(reverse).reverse().toString();
     }
 
-    public static void findSymbolOccurance (String name, char symbol) {
+    public static int findSymbolOccurance (String name, char symbol) {
```

Comment:
What will happen if `name == null`? Could you please add null validation for the input string only?